.travis.yml

```diff
@@ -3,9 +3,5 @@ language: ruby
 rvm:
   - 2.1.4
```

Comment:
There must be parity between all ruby versions in development, testing and production